examples/nft-d/nft.yml

```diff
@@ -1,7 +1,7 @@
 name: "NeoFS Object NFT"
 sourceurl: https://github.com/nspcc-dev/neo-go/
-supportedstandards: ["NEP-11"]
-safemethods: ["balanceOf", "decimals", "symbol", "totalSupply", "tokensOf", "ownerOf", "properties", "tokens"]
+supportedstandards: ["NEP-11", "NEP-24"]
+safemethods: ["balanceOf", "decimals", "symbol", "totalSupply", "tokensOf", "ownerOf", "properties", "tokens", "royaltyInfo"]
```

Comment:
Let's extend documentation for both nft-d and nft-nd: mention that these tokens also implement NEP24 standard.
https://github.com/nspcc-dev/neo-go/blob/master/examples/README.md#examples-structure